envs/example/ci-full/group_vars/all.yml

```diff
@@ -0,0 +1,15 @@
+---
+stack_env: example_ci_full
+floating_ip: 172.16.0.100
+
+xtradb:
+  sst_auth_password: asdf
+
+percona:
+  replication: False
+
+ceilometer:
+  enabled: True
+  logging:
+    debug: True
+    verbose: True
```

Comment:
Actually, the content of that all.yml will be overwritten by test/setup script during runtime, I just copied the original content of that file from existing envs/example/ci/ folder. Anyway, it makes sense to refine the static file to reflect it's final content as much as possible, I will update that file.